airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/metadata.py

```diff
@@ -149,8 +149,33 @@ def _run(self, connector: Connector) -> CheckResult:
         )
 
 
+class CheckConnectorMaxSecondsBetweenMessagesValue(MetadataCheck):
+    name = "Certified source connector must have a value filled out for maxSecondsBetweenMessages in metadata"
+    description = f"Certified source connectors must have a value filled out for maxSecondsBetweenMessages in metadata. This value represents the maximum number of seconds we could expect between messages for API connectors. And it's used by platform to tune connectors heartbeat timeout. The value must be set in the 'data' field in {consts.METADATA_FILE_NAME}"
+    applies_to_connector_types = ["source"]
+
+    @staticmethod
+    def check_connector_certified(connector: Connector) -> bool:
+        connector_sl_value = connector.metadata.get("ab_internal", {}).get("sl", 0)
+        return connector_sl_value >= 200
```

Comment:
I don't know if certified is the same as connector_sl_value being higher than 200. 
I see what you did here — but do you really want that criteria vs perhaps this check should only run on connectors that are published to cloud registry instead? @alafanechere has a method to check for whether connector is in registry in a neighboring PR somewhere.